docs/creating-components.md

```diff
@@ -156,67 +156,56 @@ const CountableDiv = Countable('div');
 const CountableDivAsDataNumber = CountableAsDataNumber('div');
 ```
 
-## Adding Design-Time Customizations to Components
+## Customizing Components at Design-Time
 
-> NOTE: `factory()` is deprecated and will be removed in 2.0
+> NOTE: `factory()`, the previous experimental technique for doing this, is deprecated and will be removed in 2.0
 
-While many of our components are easily configured using properties, we've found that sometimes an app
-needs the more flexibility. Generally, this type of customization isn't variable at
-run-time and is primarily used to adjust the visual design of a component. To support this use case,
-we've introduced the `factory()` function.
+Occasionally, you'll want to modify the appearance of an Enact component, and the majority of that time, you can simply apply external styling to the outer-most element of the component, via `className` or `style`. However, what if you need to customize one of the deeper child elements?
 
-Like `hoc()`, `factory()` accepts a configuration object which is merged with the component's default
-configuration. Unlike `hoc()`, `factory()` only supports a known set of configuration keys rather
-than an arbitrary set. Currently, the only supported key is `css`, which allows authors to provide
-a custom CSS class name map which is merged with the component's CSS class name map. This feature
-allows for overriding the style of particular children within the component's internal
-hierarchy.
+We've got you covered! In Enact 2.0 we've added a built-in theming capability to make this significantly easier and even safer. Now, components built with the `kind` feature can use the `publicClassNames` key in their `styles` block of their component definition.
 
-A few important notes about this feature:
+The `publicClassNames` key allows a component to define an array of CSS class names that will be available for a component consumer to add styling to. **NOTE: GO INTO MORE DETAIL AND LINK TO OTHER PAGE** All components in `@enact/ui` and many in `@enact/moonstone` and other themes have already been imbued with this feature.
 
-* This is a recent addition to Enact and has not been widely adopted by framework components yet
-* In order to promote UX consistency, not every class will be customizable
-* We do not yet have a means to document which classes may be customized
+Use the theming system is as easy as importing your CSS/LESS file and passing it to the `css` prop on the component you want to customize. The class names defined in your CSS file that match the published class names of the target component will be applied directly to the internal elements of the component. They will be applied in addition to the existing class names, not in-leu of, so you can simply add your customizations, rather than repeat the existing styling.
```

Comment:
Was this supposed to be "Use [of] the.. "?   'in lieu'

Reply:
"Using", and yes. I always misspell "leiu".